Sources/DDGSync/internal/Crypter.swift

```diff
@@ -23,15 +23,23 @@ struct Crypter: CryptingInternal {
 
     let secureStore: SecureStoring
 
-    func encryptAndBase64Encode(_ value: String, using secretKey: Data?) throws -> String {
-        var encryptionKey: [UInt8] = secretKey?.safeBytes ?? []
-        if encryptionKey.isEmpty {
-            guard let account = try secureStore.account() else {
-                throw SyncError.accountNotFound
-            }
-            encryptionKey = account.secretKey.safeBytes
+    func fetchSecretKey() throws -> Data {
+        guard let account = try secureStore.account() else {
+            throw SyncError.accountNotFound
```

Comment:
Side note, that we've already spoke about some time ago: we don't seem to have solid guidelines around catching&re-throwing errors, I think we should sync on how to approach that so underlying error info is not lost.